doc/whatsnew/2/2.15/index.rst

```diff
@@ -14,4 +14,12 @@ Summary -- Release highlights
 * We improved ``pylint``'s handling of namespace packages. More packages should be
   linted without resorting to using the ``-recursive=y`` option.
 
+Other Changes
+=============
+
+
+* ``bad-exception-context`` has been renamed to ``bad-exception-cause`` as it is about the cause and not the context.
+
+  Closes #3694
+
```

Comment:
Sorry you've been caught in a change of how we do thing, we added towncrier for the changelog, the instruction are in the pull request template but you already created yours so you can't see them:

- [ ] Create a news fragment with `towncrier create <IssueNumber>.<type>` which will be
  included in the changelog. `<type>` can be one of: new_check, removed_check, extension,
  false_positive, false_negative, bugfix, other, internal. If necessary you can write
  details or offer examples on how the new change is supposed to work.

Reply:
Okay I didn't know that, so I created the file 3694.other manually after seeing the pipeline failed. Could there be a problem if the file was created manually and should I generate it with towncrier instead ?